grails-app/controllers/org/pih/warehouse/api/ApiController.groovy

```diff
@@ -177,7 +177,7 @@ class ApiController {
                 supportedActivities.contains(ActivityCode.AUTOSAVE.name())
         def defaultLocale = new Locale(grailsApplication.config.openboxes.locale.defaultLocale ?: "en")
         def supportedLocales = locales.collect {
-            def defaultName = LocalizationUtil.getLocale(it)?.getDisplayName(currentLocale ?: defaultLocale)
+            def defaultName = StringUtils.parseLocaleString(it)?.getDisplayName(currentLocale ?: defaultLocale)
```

Comment:
I wouldn't have been mad if you kept LocalizationUtil.getLocale(String) and implemented the method using StringUtils.parseLocaleString().

I'm guessing I was the original author of this line and I'm now looking at it like ... what in the hell. 
```
def defaultName = StringUtils.parseLocaleString(it)?.getDisplayName(currentLocale ?: defaultLocale)
```

Oh right, I think we were trying to translate the display name of the locale as it would look in that locale. For the Locale selector in the footer. That's right. 

* Acholi 
* عربى 
* Deutsche 
* English
* Español
* Suomi
* Français
* Kreyò
* lItaliano
* Português
* Chinese